pycqed/measurement/sweep_points.py

```diff
@@ -270,44 +270,59 @@ def get_meas_obj_sweep_points_map(self, measurement_objects):
 
         If the sweep dimension has more than one sweep parameter name (dict with
         several keys), then:
-            - first tries to add to the list for each mobj only those sweep
+            - first adds to the list for each mobj only those sweep
             param names that contain the mobj_name.
-            - If it can't find the mobj_name in the sweep param name, assumes
-            there is only one param per mobj in each sweep dimension, and that
-            the order of params in each sweep dimension corresponds to the
-            order of keys in keys_list.
-            I.e. key_i in sweep_points[0] contains the sweep information for
-            measured_objects[i].
-
-        :param measured_objects: list of strings to be used as keys in the
-            returned dictionary. These are the measured object names
+            - if some parameters remain that do not contain any of the
+            measurement_objects names, it is assumed that all
+            measurement_objects used them so they will be added for each mobj
+
+        :param measurement_objects: list of strings to be used as keys in the
+            returned dictionary. These are the measured object names.
+            Can also be list of QuDev_transmon instanced in which case
```

Comment:
The comment can be written in a more general way: in the future, we might have other mobjs that have a name attribute.

Reply:
I changed this now